.travis.yml

```diff
@@ -1,19 +1,19 @@
-group: edge
+jobs:
+  include:
+    - stage: Tests
+      name: 'Kuzzle GraphQL Plugin Tests'
+      language: node_js
+      node_js: 10
+      cache:
+        directories:
+          - ~/.npm
+          - ~/.cache
+          - node_modules
 
-services:
-  - docker
+      install:
+        - npm install --silent --unsafe-perm
```

Comment:
Why are you running the command twice?

Reply:
I based myself on the [advanced-boilerplate/travis.yml](https://github.com/kuzzleio/kuzzle-plugin-advanced-boilerplate/blob/master/.travis.yml ) thinking that making a full install could also test the actual package installation instead of only installing dev dependencies. I removed it.